cli/cdp.rs

```diff
@@ -18,14 +17,6 @@ pub struct AwaitPromiseArgs {
   pub generate_preview: Option<bool>,
 }
 
-/// <https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#method-awaitPromise>
-#[derive(Debug, Clone, Deserialize)]
-#[serde(rename_all = "camelCase")]
-pub struct AwaitPromiseResponse {
```

Comment:
@crowlKats I hope removing these is fine.

Reply:
Should be fine, originally implemented these for @bartlomieju , but I think it's fine to get rid